pcdsdevices/mv_interface.py

```diff
@@ -77,11 +84,86 @@ def camonitor(self):
         Updates current position of the motor.
         """
         try:
-            while True:
+            self._mov_ev.clear()
+            while not self._mov_ev.is_set():
                 print("\r {0:4f}".format(self.position), end=" ")
-                time.sleep(0.1)
+                self._mov_ev.wait(0.1)
         except KeyboardInterrupt:
             pass
+        finally:
+            self._mov_ev.clear()
+
+    def _stop_monitor(self):
+        self._mov_ev.set()
+
+    def Tweak_base(*args):
```

Comment:
another convention: method names are supposed to be lowercase

Reply:
actually one more thing, this shouldn't be attached to the `MvInterface` class, I think it should float near the bottom of this file as a standalone function so that we can do something like:
```
from mv_interface import tweak_base
```
in another file